Dockerfile-elm

```diff
@@ -3,7 +3,9 @@ COPY elm-package.json elm-package.json
 COPY package.json package.json
 COPY gulpfile.js gulpfile.js
 COPY jarbas jarbas
+COPY contrib/copyassets.sh copyassets.sh
+RUN chmod +x copyassets.sh
 RUN npm install
 VOLUME /assets
 RUN npm run assets
-CMD ["cp", "-r", "jarbas/frontend/static/*", "/assets"]
```

Comment:
I really don't know why this didn't work :( Shame! but I copied to script to avoid that problem.